packages/backend-modules/republik/lib/address.js

```diff
@@ -1,17 +1,39 @@
-const insertAddress = (address, pgdb) => {
+const MAX_ADDRESS_NAME_LENGTH = 70
+
+const ensureStringLengthForAddress = (prop, maxLength, message, t) => {
+  if (prop.length > maxLength) {
+    throw new Error(t(message))
+  }
+}
+
+const insertAddress = (address, pgdb, t) => {
   if (!address) {
     return
   }
 
+  ensureStringLengthForAddress(
+    address.name,
+    MAX_ADDRESS_NAME_LENGTH,
+    'address/name/tooLong',
+    t,
+  )
```

Comment:
apps/www/components/Account/AddressForm.js validates `line1` and `city` – but API does not just yet. Since API is responsible for proper data, would validate these inputs before inserting or updating.